src/coreclr/inc/corinfo.h

```diff
@@ -1726,8 +1726,10 @@ struct CORINFO_FIELD_INFO
 
 struct CORINFO_THREAD_STATIC_BLOCKS_INFO
 {
-    CORINFO_CONST_LOOKUP tlsIndex;
-    uint32_t offsetOfThreadLocalStoragePointer;
+    CORINFO_CONST_LOOKUP tlsIndex;              // windows specific
+    size_t tlsGetAddrFtnPtr;                    // linux/x64 specific - address of __tls_get_addr() function
+    size_t tlsIndexObject;                      // linux/x64 specific - address of tls_index object
```

Comment:
The JIT/EE interface uses `void*` for things that are addresses. I do not think there is any existing place that uses `size_t` to pass an address. These two should be `void*` for consistency.